src/components/entity/state-info.ts

```diff
@@ -24,13 +24,15 @@ class StateInfo extends LitElement {
       return html``;
     }
 
+    const name = computeStateName(this.stateObj);
+
     return html`<state-badge
         .stateObj=${this.stateObj}
         .stateColor=${true}
       ></state-badge>
       <div class="info">
-        <div class="name" .inDialog=${this.inDialog}>
-          ${computeStateName(this.stateObj)}
+        <div class="name" .title=${name} .inDialog=${this.inDialog}>
+          ${name}
```

Comment:
This is a bit useless, if the text is already displayed on-screen it should not also be there in a title.

Reply:
But the rendered text might be cutoff, which is the reason for adding titles/tooltips, since then the user can seer the full length name.